spec/controllers/cases/ico_controller_spec.rb

```diff
@@ -357,12 +357,12 @@ def new_linked_cases_for_request(additional_params = {})
         end
 
         it "doesn't allow linking of case that isn't an FOI or SAR" do
```

Comment:
Sorry, getting confused. Does the description of this spec still match what it does?

Reply:
sorry, forgot to change this description 😳 , let me change it